### PR TITLE
add arguments to compareplot

### PR DESCRIPTION
**Pull request by aloctavodia**
* add arguments to enable/disable elements in the plot
* add `plot_kwargs` to change plot style
* small aesthetics tweaks

### CODE REVIEW

File: pymc3/plots/compareplot.py [hunk 1]
```diff
@@ -11,9 +12,22 @@ def compareplot(comp_df, ax=None):
...
     comp_df: DataFrame
-        The result of the pm.compare() function
+        the result of the `pm.compare()` function
+    insample : bool
+        plot the in-sample deviance, that is the value of the IC without the
+        penalization given by the effective number of parameters (pIC).
+        Defaults to True
+    se : bool
+        plot the standard error of the IC estimate. Defaults to True
+    dse : bool
```
Comment (fonnesbeck) [inline pymc3/plots/compareplot.py:22]: `diff_se` (or similar) would be clearer

File: pymc3/plots/compareplot.py [hunk 2]
```diff
@@ -11,9 +12,22 @@ def compareplot(comp_df, ax=None):
     ----------
 
     comp_df: DataFrame
-        The result of the pm.compare() function
+        the result of the `pm.compare()` function
+    insample : bool
```
Comment (fonnesbeck) [inline pymc3/plots/compareplot.py]: `insample_dev` would be clearer

Comment (aloctavodia) [reply pymc3/plots/compareplot.py]: ok

File: pymc3/plots/compareplot.py (hunk 1 above)
Comment (aloctavodia) [reply pymc3/plots/compareplot.py:22]: I'm using `se` and `dse` because those names are the columns names in `comp_df`. Still want me to change it?

Comment (fonnesbeck) [reply pymc3/plots/compareplot.py:22]: These are only suggestions. Feel free to ignore if they don't make sense.

**Pull request merged**